clients/samples/gemm/sample_hipblaslt_gemm_dgelu_bgrad.cpp

```diff
@@ -0,0 +1,234 @@
+            matC, HIPBLASLT_MATRIX_LAYOUT_BATCH_COUNT, &batch_count, sizeof(batch_count)));
+        CHECK_HIPBLASLT_ERROR(hipblasLtMatrixLayoutSetAttribute(
+            matC, HIPBLASLT_MATRIX_LAYOUT_STRIDED_BATCH_OFFSET, &stride_c, sizeof(stride_c)));
+        CHECK_HIPBLASLT_ERROR(hipblasLtMatrixLayoutSetAttribute(
+            matD, HIPBLASLT_MATRIX_LAYOUT_BATCH_COUNT, &batch_count, sizeof(batch_count)));
+        CHECK_HIPBLASLT_ERROR(hipblasLtMatrixLayoutSetAttribute(
+            matD, HIPBLASLT_MATRIX_LAYOUT_STRIDED_BATCH_OFFSET, &stride_d, sizeof(stride_d)));
+    }
+
+    hipblasLtMatmulDesc_t matmul;
+    CHECK_HIPBLASLT_ERROR(hipblasLtMatmulDescCreate(&matmul, HIPBLAS_COMPUTE_32F, HIP_R_32F));
+    CHECK_HIPBLASLT_ERROR(hipblasLtMatmulDescSetAttribute(
+        matmul, HIPBLASLT_MATMUL_DESC_TRANSA, &trans_a, sizeof(int32_t)));
+    CHECK_HIPBLASLT_ERROR(hipblasLtMatmulDescSetAttribute(
+        matmul, HIPBLASLT_MATMUL_DESC_TRANSB, &trans_b, sizeof(int32_t)));
+
+    hipblasLtEpilogue_t epilogue = HIPBLASLT_EPILOGUE_DGELU_BGRAD;
+    CHECK_HIPBLASLT_ERROR(hipblasLtMatmulDescSetAttribute(
+        matmul, HIPBLASLT_MATMUL_DESC_EPILOGUE, &epilogue, sizeof(epilogue)));
+
+    // Set auxiliary pointer, this is an input buffer
+    std::vector<hipblasLtHalf> h_aux_buffer(
+        m * n, static_cast<hipblasLtHalf>(1.0)); // Example values, adjust as needed
+    void* d_aux_buffer;
+    CHECK_HIP_ERROR(hipMalloc(&d_aux_buffer, m * n * sizeof(hipblasLtHalf)));
+    CHECK_HIP_ERROR(hipMemcpy(
+        d_aux_buffer, h_aux_buffer.data(), m * n * sizeof(hipblasLtHalf), hipMemcpyHostToDevice));
+    CHECK_HIPBLASLT_ERROR(hipblasLtMatmulDescSetAttribute(
+        matmul, HIPBLASLT_MATMUL_DESC_EPILOGUE_AUX_POINTER, &d_aux_buffer, sizeof(d_aux_buffer)));
+
+    // Set auxiliary leading dimension
+    const int64_t aux_ld = m;
+    CHECK_HIPBLASLT_ERROR(hipblasLtMatmulDescSetAttribute(
+        matmul, HIPBLASLT_MATMUL_DESC_EPILOGUE_AUX_LD, &aux_ld, sizeof(aux_ld)));
+
+    // Set Epilogue Aux Batch Stride
+    const int64_t aux_batch_stride = m * n;
+    CHECK_HIPBLASLT_ERROR(
+        hipblasLtMatmulDescSetAttribute(matmul,
+                                        HIPBLASLT_MATMUL_DESC_EPILOGUE_AUX_BATCH_STRIDE,
+                                        &aux_batch_stride,
+                                        sizeof(aux_batch_stride)));
+
+    // Set Desc Bias Data Type
+    int32_t bias_data_type = HIP_R_16F;
+    CHECK_HIPBLASLT_ERROR(hipblasLtMatmulDescSetAttribute(
+        matmul, HIPBLASLT_MATMUL_DESC_BIAS_DATA_TYPE, &bias_data_type, sizeof(bias_data_type)));
+
+    // Allocate and the bias tensor, the bias gradient will be stored in d_bias buffer
+    std::vector<hipblasLtHalf> h_bias(m, 0.0f);
+    void*                      d_bias;
+    CHECK_HIP_ERROR(hipMalloc(&d_bias, m * sizeof(hipblasLtHalf)));
+    CHECK_HIP_ERROR(
+        hipMemcpy(d_bias, h_bias.data(), m * sizeof(hipblasLtHalf), hipMemcpyHostToDevice));
```

Comment:
h_bias and hipMemcpy are not necessary

Reply:
Done modification and validation.
Please take a look, thx!